pkg/distro/fedora/distro_test.go

```diff
@@ -252,6 +252,7 @@ func TestImageType_BuildPackages(t *testing.T) {
 	buildPackages := map[string][]string{
 		"x86_64":  x8664BuildPackages,
 		"aarch64": aarch64BuildPackages,
+		"ppc64le": []string{},
```

Comment:
nitpick: this is redundant, since the map will return the empty type for non-existent keys.